Editor/Halodi.PackageRegistryManager.Editor.asmdef

```diff
@@ -11,9 +11,9 @@
     "allowUnsafeCode": false,
     "overrideReferences": true,
     "precompiledReferences": [
-        "860BB79D.Tomlyn.dll",
```

Comment:
does changing the name also rerecate collision from #25 ?

Reply:
No idea, but as said in the previous comment I need to make this change otherwise the Entities package will not work.

Is there a way to test if #16 is not a problem anymore?